rotkehlchen/db/filtering.py

```diff
@@ -852,13 +852,9 @@ def make(
                 values=[x.serialize_for_db() for x in excluded_locations],
                 operator='NOT IN',
             ))
-        if location_labels is not None:
-            filters.append(DBMultiStringFilter(
-                and_op=True,
-                column='location_label',
-                values=location_labels,
-                operator='IN',
-            ))
+        if location_labels is not None:  # Matches events where these addresses appear as either source or destination  # noqa: E501
```

Comment:
come on, when adjusting code don't forget to move comments. Move that, or adjust and move to the docstring of the other class.